examples/densenet_pytorch/densenet_service.py

```diff
@@ -3,21 +3,12 @@
 
 
 import torch
-import time
 import json
-import copy
 import numpy as np
-import PIL
 from PIL import Image
-from collections import OrderedDict
 import torch
```

Comment:
While we're here, may as well remove this redundant line as well.

Reply:
I removed the other `import torch` higher up line 5